### PR TITLE
LEARNER-3971 Test - iOS - Automate Whats New Screen

### PR DESCRIPTION
**Pull request by nedo**
* Verify that Whats New screen has following contents,
** "Version <released version> New Features" as heading
** Cross-icon
** Gallery of the New features, having following contents in each page,
***  New Feature's Icon
*** New Feature's Title
*** New Feature's Details
*** "Done" button at end
* Verify that following values/strings/labels are correct,
** App Version
** Feature Name/Title
** Feature Details
* Verify that on tapping cross-icon will load Main Dashboard screen
* Verify user should be able to navigate between features
* Verify on tapping "Done" will load Main Dashboard screen
* Verify after re-login with same user "Whats New" screen will not be visible

### CODE REVIEW

**Comment by nedo** [comment]
@raeeschachar PR is ready for review

cc @staubina

File: ios/pages/ios_whats_new.py [hunk 1]
```diff
@@ -107,6 +97,42 @@ def exit_features(self):
...
+            horizontal_start_point,
+            horizontal_end_point,
+            vertical_start_point,
+            vertical_end_point
+        ))
+
+        self.driver.flick(horizontal_start_point, vertical_start_point, horizontal_end_point,
+                          vertical_end_point)
+
+        return self.get_done_button()
```
Comment (raeeschachar) [inline ios/pages/ios_whats_new.py]: Empty line at the end of file

File: ios/pages/ios_whats_new.py [hunk 2]
```diff
@@ -107,6 +97,42 @@ def exit_features(self):
...
+
+        horizontal_start_point = int(element_x_position + screen_width)
+        vertical_start_point = int(element_x_position + screen_height / 2)
+        horizontal_end_point = int(element_x_position + screen_width / 4)
+        vertical_end_point = int(element_x_position + screen_height / 2)
+
+        self.log.info('horizontal_start_point {} - vertical_start_point {} - '
+                      'horizontal_end_point {} - vertical_end_point {} '.format(
+            horizontal_start_point,
+            horizontal_end_point,
```
Comment (raeeschachar) [inline ios/pages/ios_whats_new.py]: I see pep8 indentation issues here, please check

File: ios/tests/test_ios_whats_new.py [hunk 3]
```diff
@@ -34,21 +41,72 @@ def test_validate_ui_elements(self, set_capabilities, setup_logging):
...
+            assert True
 
-        assert ios_whats_new_page.get_title_textview().text == strings.WHATS_NEW_IOS_SCREEN_TITLE
-        assert ios_whats_new_page.get_close_button().text == strings.WHATS_NEW_CLOSE
-        assert ios_whats_new_page.get_main_image()
-        assert ios_whats_new_page.get_feature_title_textview().text == strings.WHATS_NEW_FEATURE_TITLE
-        assert ios_whats_new_page.get_feature_details().text == strings.WHATS_NEW_FEATURE_DETAILS
-        assert ios_whats_new_page.get_done_button().text == strings.WHATS_NEW_DONE
 
     def test_close_features_screen_smoke(self, set_capabilities, setup_logging):
```
Comment (raeeschachar) [inline ios/tests/test_ios_whats_new.py]: Too many blank lines

File: ios/pages/ios_whats_new.py (hunk 1 above)
Comment (nedo) [reply ios/pages/ios_whats_new.py]: done

File: ios/pages/ios_whats_new.py (hunk 2 above)
Comment (nedo) [reply ios/pages/ios_whats_new.py]: done

File: ios/tests/test_ios_whats_new.py (hunk 3 above)
Comment (nedo) [reply ios/tests/test_ios_whats_new.py]: done

**Comment by nedo** [comment]
@staubina PR is ready for Review. 


Regards,
Naveed

File: README.md [hunk 4]
```diff
@@ -25,21 +25,21 @@ Automated testing for edX Android and iOS mobile applications.
 
 - You will need to configure following system level Environment Variables to run tests against
 
-    - set 'Android' to execute test cases on Android or 'iOS' to execute on iOS
+    - set `Android' to execute test cases on Android or 'iOS' to execute on iOS
 
-          `export TARGET_ENVIRONMENT = 'Android'`
+          export TARGET_ENVIRONMENT = 'Android
```
Comment (staubina) [inline README.md]: Did you mean to remove the quote at the end of Android?

**Comment by nedo** [comment]
@staubina I just changed a typo in ReadMe file, can you please have a look on last Pass ? 

Regards,
Naveed

File: README.md (hunk 4 above)
Comment (nedo) [reply README.md]: Done, basically there were 2 quoted at end, I just removed only 1, and fixed the type.

**Pull request merged**